README.md

```diff
@@ -56,10 +56,10 @@ you would need to override the systemd service with the following setting:
 
 ```conf
 [Service]
-Environment="MULTIPASS_BLUEPRINTS_URL=https://github.com/USERNAME/multipass-blueprints/archive/refs/heads/BRANCH_NAME.zip"
+Environment="MULTIPASS_BLUEPRINTS_URL=https://github.com/canonical/multipass-blueprints/archive/refs/heads/<BRANCH_NAME>.zip"
```

Comment:
This wouldn't be valid for a fork (which a contributor without write access to the repository needs to use).

Reply:
Indeed, I missed the context a bit. 
USERNAME can mislead to the actual github user name of ours for the folk in Canonical. Meanwhile, USERNAME/organization_name seems to be the right name for that part of the URL. So I also do not have a good way to disambiguate this. Maybe I can change it back and just add<> to emphasize it is a variable?

Reply:
Right. How about `<REPO_OWNER>`?

Reply:
Sounds better.